spec/services/create_escort_spec.rb

```diff
@@ -0,0 +1,116 @@
+    let!(:existent_escort) { create(:escort, prison_number: prison_number, detainee: detainee, move: move) }
+
+    it 'creates a clone of the most recent escort' do
+      expect(Escort.where(prison_number: prison_number).count).to eq(1)
+      escort = described_class.call(prison_number: prison_number)
+      expect(Escort.where(prison_number: prison_number).count).to eq(2)
+      expect_detainee_to_be_cloned(existent_escort, escort)
+      expect_risk_assessment_to_be_cloned(existent_escort, escort)
+      expect_healthcare_assessment_to_be_cloned(existent_escort, escort)
+      expect_offences_to_be_cloned(existent_escort, escort)
+      expect_moves_to_be_cloned(existent_escort, escort)
+    end
+  end
+
+  def expect_detainee_to_be_cloned(existent_escort, new_escort)
+    detainee_attributes = existent_escort.detainee.attributes.except(*%w(id escort_id created_at updated_at))
+    expect(new_escort.detainee.id).not_to eq(existent_escort.detainee.id)
+    expect(new_escort.detainee).to have_attributes(detainee_attributes)
+  end
+
+  def expect_risk_assessment_to_be_cloned(existent_escort, new_escort)
+    risk_attributes = existent_escort.risk.attributes.except(*except_assessment_attributes)
+    expect(new_escort.risk.id).not_to eq(existent_escort.risk.id)
+    expect(new_escort.risk).to have_attributes(risk_attributes)
+    expect(new_escort.risk.status).to eq('needs_review')
+  end
+
+  def expect_healthcare_assessment_to_be_cloned(existent_escort, new_escort)
+    healthcare_attributes = existent_escort.healthcare.attributes.except(*except_assessment_attributes)
+    expect(new_escort.healthcare.id).not_to eq(existent_escort.healthcare.id)
+    expect(new_escort.healthcare).to have_attributes(healthcare_attributes)
+    expect(new_escort.healthcare.status).to eq('needs_review')
+
+    expected_medications_attributes = existent_escort.healthcare.medications.map { |m| m.attributes.except(*except_medication_attributes) }
+    medications_attributes = new_escort.healthcare.medications.map { |m| m.attributes.except(*except_medication_attributes) }
+    expected_medications_attributes.each do |expected_attributes|
+      expect(medications_attributes).to include(expected_attributes)
+    end
+  end
+
+  def expect_offences_to_be_cloned(existent_escort, new_escort)
+    offences_attributes = existent_escort.offences.attributes.except(*except_offences_attributes)
+    expect(new_escort.offences.id).not_to eq(existent_escort.offences.id)
+    expect(new_escort.offences).to have_attributes(offences_attributes)
+    expect(new_escort.offences.status).to eq('needs_review')
+
+    expected_current_offences_attributes = existent_escort.offences.current_offences.map { |co| co.attributes.except(*except_current_offences_attributes) }
+    current_offences_attributes = new_escort.offences.current_offences.map { |co| co.attributes.except(*except_current_offences_attributes) }
+    expected_current_offences_attributes.each do |expected_attributes|
```

Comment:
Looks like a typo. The `current_offences_attributes` needs to be passed to the expect method in order to return an object that responds to `to`.